platform/feature-api/api/src/pod-api/index.ts

```diff
@@ -6,7 +6,7 @@
  */
 
 export * from "./api";
+export * from "./default";
 export * from "./fs";
 export * from "./feature";
-export * from "./default";
 export * from "./spec";
```

Comment:
Why not export `mock-pod` too here? That way you don't have to hard code paths including the `dist` directory in code outside this package that uses it.

Reply:
Because of the dependencies. IIRC, memfs pulls in lots of node dependencies. Also, again, want to keep it separate as a test-only thing.
Alternative to the hard-coded paths would be to bundle it independently, but I need to see if it causes trouble with the downstream dependencies.